src/main/java/com/kneelawk/extramodintegrations/tconstruct/recipe/melting/MeltingEmiRecipe.java

```diff
@@ -32,18 +38,23 @@ public static MeltingEmiRecipe of(MeltingRecipe recipe) {
         } else {
             id = recipe.getId();
         }
-        return new MeltingEmiRecipe(recipe, id);
+        return new MeltingEmiRecipe(recipe, id, Config.COMMON.smelteryOreRate);
```

Comment:
Using the config directly like this feels like it's short-cutting something. Is this what the REI plugin does?

Reply:
Yes, REI does it the same way.

https://github.com/Alpha-s-Stuff/TinkersConstruct/blob/0c35d0fd41c0386f11daace0230add3f8277ccbf/src/main/java/slimeknights/tconstruct/plugin/rei/melting/MeltingCategory.java#L139-L140